cob_obstacle_distance/include/cob_obstacle_distance/distance_manager.hpp

```diff
@@ -99,6 +107,11 @@ class DistanceManager
 
         ~DistanceManager();
 
+        inline const std::string getRootFrame() const
+        {
+            return this->root_frame_;
+        }
```

Comment:
Param name = root_frame -> Therefore renamed to root_frame_id because exactly that's what it is used for.